fxa-client/src/lib.rs

```diff
@@ -55,7 +57,8 @@ pub mod errors;
 mod http_client;
 #[cfg(feature = "browserid")]
 mod login_sm;
-mod oauth;
+// mod oauth; // We might still use this someday, let's keep the code around.
```

Comment:
👀  nit: maybe just delete it

Reply:
BUT IT TOOK ME SO LONG TO CONVERT IT FROM JS. Yeah I should probably do that, it's still in the git history anyway.

Reply:
+1 to not having any rust code that isn't actually part of the crate.